src/main/resources/io/viash/platforms/nextflow/VDSL3Helper.nf

```diff
@@ -663,6 +671,14 @@ def processFactory(Map processArgs) {
   |export VIASH_TEMP="\\\$VIASH_META_TEMP_DIR"
   |export TEMP_DIR="\\\$VIASH_META_TEMP_DIR"
   |
+  |# create output dirs if need be
+  |function mkdir_parent {
+  |  for file in "\\\$@"; do 
+  |    mkdir -p "\\\$(dirname \\\$file)"
```

Comment:
I think this will give problems when `$file` contains whitespaces.
Perhaps update to:
```bash
mkdir -p "\\\$(dirname -- "\\\$file")"
```
The `--` allows this to work when the file starts with a dash.
Edit: Apparently there are even better solutions, but they are quite elaborate... 
See https://unix.stackexchange.com/a/118438

Reply:
Thanks for catching this! I added some quotes and a unit test.